dashboard/test/ui/README.md

```diff
@@ -47,16 +47,15 @@ You can find the values for these settings in your saucelabs account settings (`
 
 If you want to run tests on Sauce Labs against localhost you need to set up your tunnel:
 
-#### Latest version of Sauce Connect Proxy CLI (5.1.0)
+#### Latest version of Sauce Connect Proxy CLI (5.2.1)
 1. Login to Sauce Labs and download the [tunnel](https://app.saucelabs.com/tunnels).
-2. Uncomment and fill out the values for the "saucelabs_" properties in `locals.yml`
-   - `saucelabs_tunnel_name` can be an arbitrary name, but it needs to match what you pass as an argument to `sc run...`
-3. (Re)start your dashboard-server `./bin/dashboard-server`.
-4. Start the sauce labs tunnel
-    - `sc run -u <saucelabs_username> -k <saucelabs_authkey> -r us-west --tunnel-name <saucelabs_tunnel_name>`
-5. Run your UI test
+2. (Re)start your dashboard-server `./bin/dashboard-server`.
+3. Start the sauce labs tunnel
+    - `sc run -u <saucelabs_username> -k <saucelabs_authkey> -r us-west --tunnel-name cdo-tunnel`
+4. Run your UI test
```

Comment:
simplify docs by suggesting a default tunnel name `cdo-tunnel`, this matches drone, and reduces variability on the commands each developer runs